tests/functional/plugins/importexport/FunctionalDoiExportTestCase.inc.php

```diff
@@ -432,7 +432,7 @@ protected function testObjectsWithoutDOICannotBeSelectedForExport($exportPages)
 		$article->setData('doiSuffix', '');
 		$articleDao->updateObject($article); // Do not use PublishedArticleDAO::updatePublishedArticle() for this, otherwise the ADODB cache flush there may cause a permission error.
 		$galleyDao = DAORegistry::getDAO('ArticleGalleyDAO'); /* @var $galleyDao ArticleGalleyDAO */
-		$galleys = $galleyDao->getBySubmissionId(1, 1);
+		$galleys = $galleyDao->getBySubmissionId(1);
```

Comment:
Rather than remove the parameter here, I opted to implement it in ArticleGalleyDAO. It's not used beyond this test but follows a pattern we use elsewhere in the system.